lib/rollup/worker.ex

```diff
@@ -0,0 +1,69 @@
+require Logger
+
+defmodule Castle.Rollup.Worker do
+  use GenServer
+
+  @jobs [
+    {Castle.Rollup.Jobs.Totals, :run, []},
+  ]
+
+  def start_link do
+    GenServer.start_link(__MODULE__, %{})
+  end
+
+  def init(state) do
+    Application.get_env(:castle, :rollup_initial_delay) |> reschedule()
+    {:ok, state}
+  end
+
+  def handle_info(:work, state) do
```

Comment:
do we need a catchall `handle_info` for potential non-`:work` messages? 
`Since any message, including the ones sent via send/2, go to handle_info/2, there is a chance unexpected messages will arrive to the server. Therefore, if we don’t define the catch-all clause, those messages could cause our registry to crash, because no clause would match. ` (https://elixir-lang.org/getting-started/mix-otp/genserver.html#call-cast-or-info)

Reply:
Good catch (all).  Will add one.